Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.h

```diff
@@ -81,6 +83,7 @@ namespace ROS2
         AZStd::string m_prefabPath;
         VisualsMaker m_visualsMaker;
         CollidersMaker m_collidersMaker;
+        SensorsMaker m_gazeboSensorsMaker;
```

Comment:
Please rename, this is based on format, not a simulator

Reply:
+1 on that.